src/app/layout/redirect-status/redirect-status.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'f8-redirect-status',
+  templateUrl: 'redirect-status.component.html'
+})
+export class RedirectStatusComponent implements OnInit {
+
+  private redirects: any = {
+    _verifyEmail: {
+      success: {
+        message: 'Your e-mail has been confirmed.',
+        secMessage: 'Thank you for validating your e-mail address. You can now continue to use Openshift.io',
+        ctaLink: '_home',
+        ctaLabel: 'home dashboard'
+      },
+      fail: {
+        message: '',
+        secMessage: 'It appears there is a problem with validating your e-mail. You can reset your e-mail on your Profile Page',
+        ctaLink: '_profile',
+        ctaLabel: 'profile'
+      }
+    }
+  };
+
+  private redirectStatus: string;
+  private redirectData: object;
```

Comment:
Please create a type for this instead of using 'object'. This will also help when it's moved to ngx-widgets as users will know what fields to expect.

Reply:
Er more specifically, when the `status.component`, which expects data of some format, is moved.